benchmarks/bench_sdbaseline.py

```diff
@@ -0,0 +1,643 @@
+                             overwrite=overwrite)
+    
+    def time_chebyshev_blfunc_no_mask(self):
+        """sdbaseline: simple successful case: blfunc = 'chebyshev', maskmode = 'list' and masklist=[] (no mask) - test002"""
+        tid = '002'
+        infile = self.infile
+        outfile = self.outroot+tid+'.ms'
+        datacolumn = 'float_data'
+        maskmode = 'list'
+        blfunc = 'chebyshev'
+        spw = '3'
+        pol = 'LL'
+        overwrite = True
+        result = sdbaseline(infile=infile, datacolumn=datacolumn,
+                             maskmode=maskmode, blfunc=blfunc,
+                             spw=spw, pol=pol, outfile=outfile,
+                             overwrite=overwrite)
+    
+    
+    def time_cspline_blfunc_no_mask(self):
+        """sdbaseline: simple successful case: blfunc = 'cspline', maskmode = 'list' and masklist=[] (no mask) - test003"""
+        print("")
+
+        tid = '003'
+        infile = self.infile
+        outfile = self.outroot+tid+'.ms'
+        datacolumn = 'float_data'
+        maskmode = 'list'
+        blfunc = 'cspline'
+        overwrite = True
+        npiece = 3
+        spw='3'
+        pol='LL'
+        result = sdbaseline(infile=infile, datacolumn=datacolumn,
+                             maskmode=maskmode, blfunc=blfunc,
+                             npiece=npiece,spw=spw,
+                             pol=pol,
+                             outfile=outfile,overwrite=overwrite)
+
+    def time_blparam_overwrite(self):
+        """sdbaseline: blparam file(s) should be overwritten when overwrite=True in fit mode - test060"""
+        tid = '060'
+        infile = self.infile
+        outfile = self.outroot+tid+'.ms'
+        overwrite = False
+        datacolumn = 'float_data'
+
+        # First run
+        sdbaseline(infile=infile, outfile=outfile, overwrite=overwrite, datacolumn=datacolumn)
+
+        shutil.rmtree(outfile)
+
+        overwrite = True
+        sdbaseline(infile=infile, outfile=outfile, overwrite=overwrite, datacolumn=datacolumn)
+
+    def time_blparam_apply_no_remove(self):
+        """sdbaseline: blparam file(s) should not be removed in apply mode - test062"""
+        tid = '062'
+        infile = self.infile
+        outfile = self.outroot+tid+'.ms'
+        overwrite = False
+        datacolumn = 'float_data'
+        blmode = 'fit'
+        blformat = ['text', 'table']
+
+        # First run
+        sdbaseline(infile=infile, outfile=outfile, overwrite=overwrite, datacolumn=datacolumn, blmode=blmode, blformat=blformat)
+
+        # Keep blparam files, and remove outfile only
+        shutil.rmtree(outfile)
+
+        # Second run (in apply mode), which must be successful
+        blmode = 'apply'
+        bltable = infile + '_blparam.bltable'
+        overwrite = True
+        sdbaseline(infile=infile, outfile=outfile, overwrite=overwrite, datacolumn=datacolumn, blmode=blmode, bltable=bltable)
```

Comment:
Is there a convenient way we could only measure a single function call, e.g., write the expected blparam text file in a setup method?